### DISCUSSION TITLE
How to show a local image on csr mode using img tag

### DISCUSSION THREAD

**Discussion by ricson-hoo** [leptos-rs/leptos#3108] [Q&A] [answered]
I have this view and I want to display a image in it. 
`view! {
        <AccessController login_required = true permission_required=vec![]>
            <Page class = "page">
                <Table class="table-zebra".to_string()
                ></Table>
                <img src="/images/test.jpg"/>
            </Page>
        </AccessController>
    }`

I have placed the image on the folder [projectRoot]/public/images/test.jpg

Cargo.toml:

`[package.metadata.leptos]
assets-dir = "public"
`
my routes:

`<Router>
            <Routes fallback=|| "Page not found.">
                <Route path=StaticSegment("") view=Home/>
                <Route path=(StaticSegment("health-reminders"), StaticSegment("all")) view=AllReminders />
                <Route path=(StaticSegment("health-reminders"), StaticSegment("templates")) view=Templates />
                <Route path=(StaticSegment("health-reminders"), StaticSegment("pending")) view=Pending />
            </Routes>
        </Router>`

I run the project using trunk serve --open 

but the image did not show up.

when I navigate to http://localhost:8080/images/test.jpg,  it shows "Page not found.", which is the fallback text of the Routes.

Am i miss something? should there be a route for static files? if so, how?

**Comment by ricson-hoo** [comment, marked as answer]
Problem solved. It turns out that I need to add this line to index.html:

`<link data-trunk rel="copy-dir" href="/public/images"/>`

**Reply by rendivs925** [reply]
it works, thanks, you saved my life...
